core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -223,8 +230,8 @@ class LogCleaner(initialConfig: CleanerConfig,
       info(s"Updating logCleanerIoMaxBytesPerSecond: $maxIoBytesPerSecond")
       throttler.updateDesiredRatePerSec(maxIoBytesPerSecond)
     }
-
-    shutdown()
+//    call shutdownCleaners() instead of shutdown to avoid unnecessary deletion of metrics
```

Comment:
Nit: perhaps indent the comment

Reply:
Thanks for the comments, applied, please take a look.